external/coreclr/project.json.template

```diff
@@ -2,16 +2,16 @@
   "frameworks": {
     "{TFM}": {
       "dependencies": {
-        "Microsoft.NETCore.Platforms": "1.2.0-beta-24721-02",
-        "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24904-03",
-        "Microsoft.NETCore.TestHost": "1.2.0-beta-24904-03",
-        "runtime.native.System.Data.SqlClient.sni": "4.4.0-beta-24814-01",
+        "Microsoft.NETCore.Platforms": "2.0.0-beta-24916-02",
```

Comment:
@ericstj Is there any way we can pull in the runtime.json directly and let nuget see it without needing to have a dependency on this package which is produced in corefx?